default/methods/share/share.py

```diff
@@ -380,16 +380,16 @@ def email_existing_user(
         if self.notify is False:
             return
 
-        subject = "Added to project " + self.project_string_id
+        subject = f"Added to project {self.project_string_id}"
 
-        message = " You have been added to: " + self.project_string_id
-        message += " as a " + str(permission_type)
+        message = f" You have been added to: {self.project_string_id}"
+        message += f" as a {str(permission_type)}"
 
         message += ". Access the project here. " + settings.URL_BASE + \
```

Comment:
I think this one is missing :)

Reply:
I used https://github.com/ikamensh/flynt to help with the conversion. Afterwards it was a lot of manual checking of all the changes to make sure the conversion was good. 

I just ran it again on this specific file, and for some reason it doesn't want to convert the one on line 388 there.

Good catch though!

Reply:
Thanks @MathijsNL